source/_layouts/page-pattern.html

```diff
@@ -0,0 +1,57 @@
+---
+layout: default
+---
+<div id="main" class="col-md-12 clearfix" role="main">
+  <article class="post">
+    <header class="page-tabs-header">
+      <div class="row">
+        <div class="col-sm-12 col-md-6 cols-nopad-right">
+          <h1>{{ page.title }}</h1>
+        </div>
+        <div class="col-sm-12 col-md-6 cols-nopad-left cols-nopad-right">
+          {% if page.code == false %}
+            {% include nav-tabs-pattern.html codetab=false %}
+          {% elsif page.design == false %}
+            {% include nav-tabs-pattern.html designtab=false %}
+          {% else %}
+            {% include nav-tabs-pattern.html %}
+          {% endif %}
+        </div>
+      </div>
+    </header>
+    <section class="post_content">
+      <div class="tab-content">
+        <div role="tabpanel" class="tab-pane active" id="overview">
+          {% capture my_include %}{% include {{page.overview}} %}{% endcapture %}
+          {{ my_include | site_replace | markdownify }}
+        </div>
+        <div role="tabpanel" class="tab-pane" id="design">
+          {% capture my_include %}{% include {{page.design}} %}{% endcapture %}
+          {{ my_include | site_replace | markdownify }}
+        </div>
+        <div role="tabpanel" class="tab-pane" id="code">
```

Comment:
IDK. I'm afraid this template is only going to work for heat map, which isn't a typical use case. If we introduce a new template, these use cases should be considered.

A good example to look at would be any of the cards (e.g., utilization trend card). There are multiple Patternfly examples here, included via templates. These particular  examples render multiple charts, which require unique IDs.

Some examples (e.g., aggregate-status-card) also include JavaScript to initialize the examples properly. Other pages include extra javaScript in the pretty print to show the required includes.